src/components/Patient/ManagePatients.tsx

```diff
@@ -804,169 +803,116 @@ export const PatientManager = () => {
-                  navigate(`/facility/${onlyAccessibleFacility.id}/patient`);
-                else if (
-                  !showAllFacilityUsers.includes(authUser.user_type) &&
-                  !authUser.home_facility_object?.id
-                )
-                  Notification.Error({
-                    msg: "Oops! No home facility found",
-                  });
-                else setShowDialog("create");
-              }}
+                  updateQuery({ is_active: "False" });
+                  return;
+                }
+
+                Notification.Warn({
+                  msg: t("select_facility_for_discharged_patients_warning"),
+                });
+                setShowDialog("list-discharged");
+              }
+            }}
+            currentTab={tabValue}
+          />
+          {!!params.facility && (
+            <ButtonV2
               className="w-full lg:w-fit"
+              id="doctor-connect-patient-button"
+              onClick={() => {
+                triggerGoal("Doctor Connect Clicked", {
+                  facilityId: qParams.facility,
+                  userId: authUser.id,
+                  page: "FacilityPatientsList",
+                });
+                setShowDoctors(true);
+              }}
             >
-              <CareIcon icon="l-plus" className="text-lg" />
-              <p id="add-patient-div" className="lg:my-[2px]">
-                Add Patient
-              </p>
+              <CareIcon icon="l-phone" className="text-lg" />
+              <p className="lg:my-[2px]">Doctor Connect</p>
             </ButtonV2>
-          </div>
-          <div className="flex w-full flex-col items-center justify-end gap-2 lg:ml-3 lg:w-fit lg:flex-row lg:gap-3">
-            <Tabs
-              tabs={[
-                { text: t("live"), value: 0 },
-                { text: t("discharged"), value: 1 },
-              ]}
-              onTabChange={(tab) => {
-                if (tab === 0) {
-                  updateQuery({ is_active: "True" });
-                } else {
-                  const id = qParams.facility || onlyAccessibleFacility?.id;
-                  if (id) {
-                    navigate(`facility/${id}/discharged-patients`);
-                    return;
-                  }
+          )}
 
-                  if (
-                    authUser.user_type === "StateAdmin" ||
-                    authUser.user_type === "StateReadOnlyAdmin"
-                  ) {
-                    updateQuery({ is_active: "False" });
-                    return;
-                  }
-
-                  Notification.Warn({
-                    msg: t("select_facility_for_discharged_patients_warning"),
-                  });
-                  setShowDialog("list-discharged");
-                }
-              }}
-              currentTab={tabValue}
-            />
-            {!!params.facility && (
+          <AdvancedFilterButton onClick={() => advancedFilter.setShow(true)} />
```

Comment:
<img width="1439" alt="image" src="https://github.com/user-attachments/assets/8f1a9b08-7e64-498f-ac27-e7ae3971a38c" />
Keep this in one line, Lets call Advanced filters, fitlers.